core/templates/pages/exploration-player-page/learner-experience/conversation-skin.component.html

```diff
@@ -1,13 +1,13 @@
 <div role="main" class="oppia-conversation-skin-animate-cards-container position-relative" *ngIf="hasFullyLoaded">
   <div class="conversation-skin-cards-container conversation-skin-animate-cards e2e-test-conversation-skin-cards-container"
-       [ngClass]="{'animate-to-two-cards': isAnimatingToTwoCards, 'animate-to-one-card': isAnimatingToOneCard}">
+       [ngClass]="{'animate-to-two-cards': conversationSkinService.isAnimatingToTwoCards, 'animate-to-one-card': conversationSkinService.isAnimatingToOneCard}">
```

Comment:
It's usually better to have as few public fields as possible in classes. It helps you in the future if you want to change implementation of any public functionality. Suppose in the future "is animating to two cards" will have to be calculated based on two fields rather than one. If you encapsulate it in a function you need to change only one place - that function. If you use the raw field you'll have to change every place that field is used.

The rule above does not apply to plain data classes with no logic inside, or to public _constants_ of any classes.

I suggest introducing a function isAnimatingToTwoCards in the service. It applies to other fields that you plan to expose.